package.json

```diff
@@ -24,17 +24,15 @@
     "Notebooks",
     "AI"
   ],
-  "tags": [
+  "keywords": [
```

Comment:
![Screenshot 2025-06-12 at 13 53 17](https://github.com/user-attachments/assets/27e895e8-3636-4243-b889-3d6779b40d5f)
I think the [docs](https://code.visualstudio.com/api/references/extension-manifest#fields) were updated and the `tags` field is not documented anymore. The tags and keywords were supposed to be merged together on the marketplace page but it appears to not be the case (see screenshot).

Having only keywords should make list of our tags very clear & follow the latest documentation